index.yaml

```diff
@@ -54,12 +54,12 @@ entries:
     version: 1.0.0
   - apiVersion: v2
     appVersion: v0.3.1
-    created: "2021-07-29T19:49:40.598929085Z"
+    created: "2022-01-20T20:01:59.529636005Z"
     description: An addon for cert-manager to sign certificates using AWS PCA
     digest: 35d9e273aaae6f7ae89f28e76da80b09330f1fc833f0404cc6256ed6a9fe08b1
     name: aws-privateca-issuer
     type: application
     urls:
-    - https://cert-manager.github.io/aws-privateca-issuer/aws-pca-issuer-0.1.2.tgz
+    - https://cert-manager.github.io/aws-privateca-issuer/aws-privateca-issuer-0.1.2.tgz
```

Comment:
Changed this endpoint to match the others, I believe it's different because of the merge that used to be happening